src/evaluation/factuality.py

```diff
@@ -133,6 +135,22 @@ def evaluate_summary(
             if not ent["in_source"]:
                 summary_eval["count_entity_extrinsic"] += 1
 
+        # If there are any extrinsic hallucinations, then keep the ratio set to
```

Comment:
Thinking a bit more about this and had an idea - perhaps ratio should be set to 0 to penalize summaries that have no extrinsic hallucinations?

Not sure if that's mathematically sound, but something to consider

Reply:
We decided not to do this and likely contextualize the main results table with some measure of average number of extrinsic entities per summary